archinstall/lib/menu/selection_menu.py

```diff
@@ -1,5 +1,7 @@
 import sys
 
+from archinstall import Translation
```

Comment:
We should fix all these imports. They should be relative from /lib/ and not import archinstall, otherwise they would use the wrong version if both installed and git cloned and used "locally"

Reply:
I've added the import for `Translation` in the __init__.py now as we do with others, is that what you meant?

Reply:
We have to do: https://github.com/archlinux/archinstall/blob/e65d9723bb72e2cdfe9935af63a4873414537943/archinstall/lib/user_interaction.py#L31

Otherwise this will fail if use as modules, git clones and other weird use cases : ) I suspect it's your IDE wanting you to import it like this because you have it installed as a module and it auto-detects this path.